model/src/application/interaction/application_command/data/resolved.rs

```diff
@@ -40,6 +40,7 @@ pub struct InteractionChannel {
 
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 pub struct InteractionMember {
+    pub communication_disabled_until: Option<Timestamp>,
```

Comment:
has it been tested that this field is present on `InteractionMember`s?

Reply:
According to the documentation, the field _should_ be present: https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-object-resolved-data-structure. But I will be sure to do some checks on that.